org.eclipse.jdt.core/model/org/eclipse/jdt/internal/core/CompilationUnit.java

```diff
@@ -490,27 +477,20 @@ public IJavaElement[] codeSelect(int offset, int length, WorkingCopyOwner workin
 	}
 }
 
-public org.eclipse.jdt.core.dom.CompilationUnit getOrBuildAST(WorkingCopyOwner workingCopyOwner, int focalPosition) throws JavaModelException {
+public org.eclipse.jdt.core.dom.CompilationUnit getOrBuildAST(WorkingCopyOwner workingCopyOwner) throws JavaModelException {
```

Comment:
I think we should currently keep the focalPosition and instead try to make use of it for eg completion.